### ISSUE TITLE
Support for sharded installations?

### ISSUE THREAD

**Issue by noordawod** [j256/ormlite-core#58]
Hi there.
I started using the library and have gotten to a point where I'm stuck. My infrastructure contains many shards (each database contains the same table names, most of the time), and what changes is the database name.
However, I cannot seem to find an easy way how to query a table from different databases. It seems that in order to use OrmLite, one needs to first select a database and then use the different methods to interact with tables: querying, inserting, updating, etc.
So, in short: How to support shards easily?
Thanks.

**Comment by noordawod** [comment]
I opened a more focused issue: #63
